curriculum/challenges/english/blocks/quiz-styling-forms/66ed9043f45ce3ece4053ebf.md

```diff
@@ -61,23 +61,23 @@ Text input fonts need to be adequately sized, and the color needs to have suffic
 
 #### --text--
 
-Why is it important to not remove the ability for users to resize a `textarea` input?
+Why is it important to provide sufficient padding inside text inputs?
```

Comment:
Questions in the quiz should be related to the earlier lessons in the same module: https://www.freecodecamp.org/learn/responsive-web-design-v9/#lecture-best-practices-for-styling-forms.

This topic is not mentioned in the lessons.